lightning/src/offers/offer.rs

```diff
@@ -344,8 +344,11 @@ macro_rules! offer_builder_methods { (
 	///
 	/// Successive calls to this method will add another blinded path. Caller is responsible for not
 	/// adding duplicate paths.
-	pub fn path($($self_mut)* $self: $self_type, path: BlindedPath) -> $return_type {
-		$self.offer.paths.get_or_insert_with(Vec::new).push(path);
+	pub fn path($($self_mut)* $self: $self_type, paths: Vec<BlindedPath>) -> $return_type {
+		let entry = $self.offer.paths.get_or_insert_with(Vec::new);
+		for path in paths {
+			entry.push(path)
+		}
```

Comment:
We probably don't want to change this, but FWIW `extend` is more idiomatic.

Reply:
Done! [f32a93a](https://github.com/lightningdevkit/rust-lightning/commit/f32a93acc60d20078c42b96dbb4466969b06f5b3)
Thanks!

Reply:
Oh, but we should definitely revert the change.